highfidelity/hifi
# Movement changes for Hand Controllers

This PR changes some default locomotion settings when using Hand Controllers.
- By default, users cannot translate with left pad / stick.
- By default, users cannot jump with right pad / stick.
- Be default, the horizontal axis of the left and right sticks turns your avatar.
- By default, pressing the down direction of either stick / pad will rotate your avatar 180 degrees.

To enable locomotion by translation and jumping (previous behavior), users can select "Advanced Movement For Hand Controllers" from the Settings Menu.

To test:
1. Load this PR and its default scripts.
2. You should not be able to walk or jump with the hand controllers. 
3. Check "Advanced Movement For Hand Controllers" in the Settings Menu.
4. You should be able to walk and jump with the controllers.
